Makefile

```diff
@@ -106,10 +106,7 @@ docs: tools
 tools: tool_rust tool_fmt tool_clippy tool_readme
 
 tool_rust:
-	@if rustup --version >/dev/null 2>&1; then \
-		echo "# Makefile # found rustup, setting override stable"; \
-		rustup override set stable; \
```

Comment:
Oh this is interesting, I think I missed this in the Lair repo when I added 1.66.1 to CI

Reply:
: / - i mean, i guess this is less necessary at the moment, since my system isn't defaulting to nightly, but it will be annoying not having it if I ever switch again.

Reply:
It really bothered me that this forces swapping out my toolchain while running unrelated commands. Especially as I'd picked a different toolchain on purpose to test the build against 1.66 which was replaced before the build actually happened. I'm fine with dockerfiles and nix files or whatever else is scoped to the project but I don't want my system state to be affected by running commands to build individual projects

Reply:
Mmm, I can see that. A convenience I'm not allowed to have, then.